README.md

```diff
@@ -18,256 +18,6 @@
-    flow=vertex_ai_job_flow,
-    name="vertex-ai-job-deployment", 
-    version=1, 
-    work_queue_name="demo",
-)
-deployment.apply()
-```
-
-Run the deployment either on the UI or through the CLI:
-```bash
-prefect deployment run vertex-ai-job-flow/vertex-ai-job-deployment
-```
-
-Visit [Prefect Deployments](https://docs.prefect.io/tutorials/deployments/) for more information about deployments.
-
-## Resources
-
-For more tips on how to use tasks and flows in a Collection, check out [Using Collections](https://docs.prefect.io/collections/usage/)!
-
-### Installation
-
-To use `prefect-gcp` and Cloud Run:
-
-```bash
-pip install prefect-gcp
-```
-
-To use Cloud Storage:
-```bash
-pip install "prefect-gcp[cloud_storage]"
-```
-
-To use BigQuery:
-
-```bash
-pip install "prefect-gcp[bigquery]"
-```
-
-To use Secret Manager:
-```bash
-pip install "prefect-gcp[secret_manager]"
-```
-
-To use Vertex AI:
-```bash
-pip install "prefect-gcp[aiplatform]"
-```
-
-A list of available blocks in `prefect-gcp` and their setup instructions can be found [here](https://prefecthq.github.io/prefect-gcp/#blocks-catalog).
-
-Requires an installation of Python 3.7+.
-
-We recommend using a Python virtual environment manager such as pipenv, conda or virtualenv.
-
-These tasks are designed to work with Prefect 2.0. For more information about how to use Prefect, please refer to the [Prefect documentation](https://docs.prefect.io/).
-
-### Feedback
-
-If you encounter any bugs while using `prefect-gcp`, feel free to open an issue in the [prefect-gcp](https://github.com/PrefectHQ/prefect-gcp) repository.
-
-If you have any questions or issues while using `prefect-gcp`, you can find help in either the [Prefect Discourse forum](https://discourse.prefect.io/) or the [Prefect Slack community](https://prefect.io/slack).
-
-Feel free to star or watch [`prefect-gcp`](https://github.com/PrefectHQ/prefect-gcp) for updates too!
-
-### Contributing
-
-If you'd like to help contribute to fix an issue or add a feature to `prefect-gcp`, please [propose changes through a pull request from a fork of the repository](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork).
-
-Here are the steps:
-
-1. [Fork the repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo#forking-a-repository)
-2. [Clone the forked repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo#cloning-your-forked-repository)
-3. Install the repository and its dependencies:
-```
-pip install -e ".[dev]"
-```
-4. Make desired changes
-5. Add tests
-6. Insert an entry to [CHANGELOG.md](https://github.com/PrefectHQ/prefect-gcp/blob/main/CHANGELOG.md)
-7. Install `pre-commit` to perform quality checks prior to commit:
-```
-pre-commit install
-```
-8. `git commit`, `git push`, and create a pull request
+Visit the full docs [here](https://PrefectHQ.github.io/prefect-gcp).
```

Comment:
Basic install might be misleading with the "extras"

Contribution I suppose we could keep it here, but do we want to duplicate it on the landing page too?

Reply:
I would recommend moving contributing off of the landing page. I think it could be it's own section in the docs and be located in the README. For installation, I would have basic installation and mention the extras with a link to the docs. I think a README with almost nothing in it would be somewhat jarring.